spec/aws_support/matchers/create_an_aws_object.rb

```diff
@@ -41,7 +41,7 @@ def match_failure_messages(recipe)
         differences += match_values_failure_messages(example.be_updated, recipe, "recipe")
 
         # Check for object existence and properties
-        resource = resource_class.new(name, nil)
+        resource = resource_class.new(name, recipe.client.run_context)
```

Comment:
It is necessary so that we can [use the provider to look up the tags](https://github.com/chef/chef-provisioning-aws/blob/tball/chef_master/spec/aws_support/matchers/have_aws_object_tags.rb#L47-L53).

I'll remove it from the non-tag matchers